index.html

```diff
@@ -9,6 +9,7 @@
     <script src="scripts/index.js"></script>
     <link href="https://unpkg.com/aos@2.3.1/dist/aos.css" rel="stylesheet">
     <script src="https://unpkg.com/aos@2.3.1/dist/aos.js"></script>
+    <link href="https://raw.githubusercontent.com/AtharvaK123/Sites-For-All.github.io/main/styles/logo.png" rel="icon" />
```

Comment:
Please remove "https://raw.githubusercontent.com/AtharvaK123/Sites-For-All.github.io/main/" for it to work.